app/assets/stylesheets/application.scss

```diff
@@ -19,3 +19,4 @@
 @import 'prism';
 @import 'components/**/*';
 @import 'responsive';
+@import 'typography'
```

Comment:
Can we move this import to just after the import for global please?

With these styles being on element selectors now, we will need to import the typography stylesheet before components I think.